lang/en.json

```
@@ -431,6 +431,7 @@
     "PerkChoiceEnvironments": "Environments",
     "PerkChoiceNone": "None",
     "PerkChoicePerks": "Perks",
+    "PerkChoiceQuantity": "Quantity of Choices",
```

```suggestion
    "PerkNumChoices": "Number of Perk Choices",
```
Wondering if this would make the field more clear?